src/site/content/en/secure/fixing-mixed-content/index.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: post
+title: Fixing mixed content
+authors:
+  - johyphenel
+  - rachelandrew
+date: 2019-09-07
+updated: 2020-09-21
+description: >
+  Find out how to fix mixed content errors on your website,
+  in order to protect users and ensure that all of your content loads.
+tags:
+  - security
+---
+
+Supporting HTTPS for your website is an important step to protecting your site and your users from attack,
+but mixed content can render that protection useless.
+Increasingly insecure mixed content will be blocked by browsers, as explained in [What is mixed content?](/what-is-mixed-content)
+
+In this guide we will demonstrate techniques and tools for fixing existing mixed content issues
+and preventing new ones from happening.
+
+## Finding mixed content by visiting your site
+
+When visiting an HTTPS page in Google Chrome,
+the browser alerts you to mixed content as errors and warnings in the JavaScript console.
+
+In [What is mixed content?](/what-is-mixed-content),
+you can find a number of examples and see how the problems are reported in DevTools.
+
+The example of [passive mixed content](https://passive-mixed-content.glitch.me/) will give the following warnings.
+
+<figure class="w-figure">
+  <img class="w-screenshot"
+      src="passive-mixed-content.jpg"
+      alt="Chrome DevTools showing the warnings displayed when mixed content is detected and upgraded">
+</figure>
+
+[Active mixed content](https://active-mixed-content.glitch.me/) is blocked and displays a warning.
+
+<figure class="w-figure">
+  <img class="w-screenshot"
+      src="active-mixed-content.jpg"
+      alt="Chrome DevTools showing the warnings displayed when active mixed content is blocked">
+</figure>
+
+If you find warnings like these on your site,
+you need to fix the `http://` URLs listed in these errors and warnings in your site's source.
```

Comment:
Maybe 'If you find warnings like these for `http://` URLs on your site,
you need to fix them in your site's source.'